src/chunk_adaptive.c

```diff
@@ -429,7 +429,9 @@ ts_calculate_chunk_interval(PG_FUNCTION_ARGS)
 	if (PG_NARGS() != CHUNK_SIZING_FUNC_NARGS)
 		elog(ERROR, "invalid number of arguments");
 
-	Assert(chunk_target_size_bytes >= 0);
+	if (chunk_target_size_bytes < 0)
+		elog(ERROR, "chunk_target_size must be positive");
+
```

Comment:
Maybe you want to use Ensure here?

Reply:
that would make it throw generate coredumps in sqlsmith runs which is what we want to avoid

Reply:
Correct, because we are running it on a `Debug` build.

Since production builds use `Release` (or `RelWithDebugInfo`) builds, there might be situations that are not captured with the current setup. It might be a good idea to run sqlsmith on `Release` (or `RelWithDebugInfo`) builds to ensure that we do not have any lingering problems here.

Regardless, I approved the pull request so from my side you can merge this as you see fit.